reark/src/test/java/io/reark/reark/data/stores/cores/ContentProviderStoreCoreBaseTest.java

```diff
@@ -29,55 +28,55 @@ public void setup() {
     }
 
     private static Observable<Integer> createSource(int numItems) {
-        return Observable.fromEmitter(publisher -> {
+        return Observable.fromPublisher(publisher -> {
```

Comment:
Wouldn't `Observable.range(1, numItems)` be better?

Reply:
It is, fixed